app/views/api/v2/host_statuses/index.json.rabl

```diff
@@ -0,0 +1,17 @@
+collection @host_statuses
+
+attributes :name, :description, :ok_total_path, :ok_owned_path, :warn_total_path,
+  :warn_owned_path, :error_total_path, :error_owned_path
+
+node :details do |presenter|
+  presenter.all_statuses.map do |status|
+    {
+      label: presenter.labels.fetch(status, "Status #{status}"),
```

Comment:
`status` is a number here, it is not very informative to the user, could we translate it to something more meaningful?

Reply:
status number is only displayed if labels are not defined, see build status labels

https://github.com/kamils-iRonin/foreman/blob/d410d067f449ec2102c4a2fec9d5431d0f05c8c6/app/models/host_status/build_status.rb#L12-L17

Reply:
Yep, I'm wondering if we want to simply hide these statuses until the labels are defined in the respective plugins.